docs/governance/intro/roadmap.mdx

```diff
@@ -367,16 +367,16 @@ Kusion Model 2022 路线规划以季度周期建立里程碑：
 - 代码质量：帮助提升模型的易用性和稳定性；
 
 ## Kusion Engine 路线规划
-Kusion Engine负责对KCL编译结果的解析和资源下发，主要包含三个部分：资源依赖解析与执行，资源State管理，多类型Runtime对接
+Kusion Engine负责对KCL编译结果的解析和资源下发，主要包含三个部分：资源依赖解析与执行，资源State管理，异构Runtime管理
```

Comment:
all: add space before and after english words

Reply:
fixed